Remove all the symbols identified in the build process as missing.
The symbols may appear in official JDK, I believe.

To see all symbols, build up to failure and run the last command with added:
    -Wl,--error-limit=0
to see all missing symbols.

Then patch the mapfile-vers via:
    for l in $(cat to_remove.txt)
    do
        echo "Removing $l ..."
        sed -i "/$l/d" mapfile-vers
    done

This patch is made with the process just above.

--- mapfile-vers.orig	2025-12-07 22:20:51.342050304 +0100
+++ mapfile-vers	2025-12-07 22:20:51.382899163 +0100
@@ -11,25 +11,19 @@
                _ZN3WTF6StringC1EPKc;
                _ZN3WTF6StringC1ESt4spanIKDsLm18446744073709551615EE;
                _ZN3WTF6StringC1ESt4spanIKDsLm18446744073709551615EE;
-               _ZN3WTF6StringC1EPKDsj;
                _ZN3WTF7RunLoop4mainEv;
                _ZN3WTF6Config25disableFreezingForTestingEv;
                _ZN3WTF7RunLoop8dispatchEONS_8FunctionIFvvEEE;
                _ZN3WTF8fastFreeEPv;
-               _ZN3WTF8LockBase8lockSlowEv;
-               _ZN3WTF8LockBase10unlockSlowEv;
                _ZN3WTF4Lock10unlockSlowEv;
                _ZN3WTF4Lock8lockSlowEv;
                _ZN3WTF8WallTime3nowEv;
-               _ZN3WTF10fastMallocEj;
                _ZN3WTF10fastMallocEm;
                _ZN3WTF10StringImpl7destroyEPS0_;
-               _ZN3WTF16fastZeroedMallocEj;
                _ZN3WTF16fastZeroedMallocEm;
                _ZN3WTF26getAndResetAccumulatedLogsEv;
                _ZNK3WTF6String4utf8ENS_14ConversionModeE;
                _ZNK14OpaqueJSString6stringEv;
-               _ZNK3WTF10StringView23underlyingStringIsValidEv;
                _ZN3WTF26setPermissionsOfConfigPageEv;
                JSCheckScriptSyntax;
                JSClassCreate;
@@ -123,7 +117,6 @@
                JSValueToStringCopy;
                JSValueUnprotect;
                _ZN3JSC6Config23enableRestrictedOptionsEv;
-               _ZN3JSC6Config25disableFreezingForTestingEv;
                _ZN3JSC16slowValidateCellEPNS_6JSCellE;
                _ZN3JSC14setNeverInlineEPK15OpaqueJSContextPK13OpaqueJSValue;
                _ZN3JSC19numberOfDFGCompilesEPK15OpaqueJSContextPK13OpaqueJSValue;
@@ -217,33 +210,6 @@
                Java_com_sun_webkit_dom_DOMImplementationImpl_createDocumentTypeImpl;
                Java_com_sun_webkit_dom_DOMImplementationImpl_createHTMLDocumentImpl;
                Java_com_sun_webkit_dom_DOMImplementationImpl_dispose;
-               Java_com_sun_webkit_dom_DOMImplementationImpl_hasFeatureImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_addRangeImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_collapseImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_collapseToEndImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_collapseToStartImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_containsNodeImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_deleteFromDocumentImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_dispose;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_emptyImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_extendImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getAnchorNodeImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getAnchorOffsetImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getBaseNodeImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getBaseOffsetImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getExtentNodeImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getExtentOffsetImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getFocusNodeImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getFocusOffsetImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getIsCollapsedImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getRangeAtImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getRangeCountImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_getTypeImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_modifyImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_removeAllRangesImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_selectAllChildrenImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_setBaseAndExtentImpl;
-               Java_com_sun_webkit_dom_DOMSelectionImpl_setPositionImpl;
                Java_com_sun_webkit_dom_DOMStringListImpl_containsImpl;
                Java_com_sun_webkit_dom_DOMStringListImpl_dispose;
                Java_com_sun_webkit_dom_DOMStringListImpl_getLengthImpl;
@@ -327,7 +293,6 @@
                Java_com_sun_webkit_dom_DOMWindowImpl_getOnresetImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_getOnresizeImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_getOnscrollImpl;
-               Java_com_sun_webkit_dom_DOMWindowImpl_getOnsearchImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_getOnseekedImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_getOnseekingImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_getOnselectImpl;
@@ -427,7 +392,6 @@
                Java_com_sun_webkit_dom_DOMWindowImpl_setOnresetImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_setOnresizeImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_setOnscrollImpl;
-               Java_com_sun_webkit_dom_DOMWindowImpl_setOnsearchImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_setOnseekedImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_setOnseekingImpl;
                Java_com_sun_webkit_dom_DOMWindowImpl_setOnselectImpl;
@@ -534,13 +498,10 @@
                Java_com_sun_webkit_dom_DocumentImpl_getOnreadystatechangeImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getOnresetImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getOnscrollImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_getOnsearchImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getOnselectImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getOnselectionchangeImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getOnselectstartImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getOnsubmitImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_getOnwebkitfullscreenchangeImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_getOnwebkitfullscreenerrorImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getOverrideStyleImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getPreferredStylesheetSetImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getReadyStateImpl;
@@ -550,7 +511,6 @@
                Java_com_sun_webkit_dom_DocumentImpl_getTitleImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getURLImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getVisibilityStateImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_getWebkitCurrentFullScreenElementImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getWebkitFullScreenKeyboardInputAllowedImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getWebkitFullscreenElementImpl;
                Java_com_sun_webkit_dom_DocumentImpl_getWebkitFullscreenEnabledImpl;
@@ -568,7 +528,6 @@
                Java_com_sun_webkit_dom_DocumentImpl_querySelectorAllImpl;
                Java_com_sun_webkit_dom_DocumentImpl_querySelectorImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setBodyImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_setCharsetImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setCookieImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setOnabortImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setOnbeforecopyImpl;
@@ -608,19 +567,14 @@
                Java_com_sun_webkit_dom_DocumentImpl_setOnreadystatechangeImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setOnresetImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setOnscrollImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_setOnsearchImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setOnselectImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setOnselectionchangeImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setOnselectstartImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setOnsubmitImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_setOnwebkitfullscreenchangeImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_setOnwebkitfullscreenerrorImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setSelectedStylesheetSetImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setTitleImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setXmlStandaloneImpl;
                Java_com_sun_webkit_dom_DocumentImpl_setXmlVersionImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_webkitCancelFullScreenImpl;
-               Java_com_sun_webkit_dom_DocumentImpl_webkitExitFullscreenImpl;
                Java_com_sun_webkit_dom_DocumentTypeImpl_getEntitiesImpl;
                Java_com_sun_webkit_dom_DocumentTypeImpl_getInternalSubsetImpl;
                Java_com_sun_webkit_dom_DocumentTypeImpl_getNameImpl;
@@ -690,12 +644,9 @@
                Java_com_sun_webkit_dom_ElementImpl_getOnpasteImpl;
                Java_com_sun_webkit_dom_ElementImpl_getOnresetImpl;
                Java_com_sun_webkit_dom_ElementImpl_getOnscrollImpl;
-               Java_com_sun_webkit_dom_ElementImpl_getOnsearchImpl;
                Java_com_sun_webkit_dom_ElementImpl_getOnselectImpl;
                Java_com_sun_webkit_dom_ElementImpl_getOnselectstartImpl;
                Java_com_sun_webkit_dom_ElementImpl_getOnsubmitImpl;
-               Java_com_sun_webkit_dom_ElementImpl_getOnwebkitfullscreenchangeImpl;
-               Java_com_sun_webkit_dom_ElementImpl_getOnwebkitfullscreenerrorImpl;
                Java_com_sun_webkit_dom_ElementImpl_getPreviousElementSiblingImpl;
                Java_com_sun_webkit_dom_ElementImpl_getScrollHeightImpl;
                Java_com_sun_webkit_dom_ElementImpl_getScrollLeftImpl;
@@ -703,7 +654,6 @@
                Java_com_sun_webkit_dom_ElementImpl_getScrollWidthImpl;
                Java_com_sun_webkit_dom_ElementImpl_getStyleImpl;
                Java_com_sun_webkit_dom_ElementImpl_getTagNameImpl;
-               Java_com_sun_webkit_dom_ElementImpl_getWebkitRegionOversetImpl;
                Java_com_sun_webkit_dom_ElementImpl_hasAttributeImpl;
                Java_com_sun_webkit_dom_ElementImpl_hasAttributeNSImpl;
                Java_com_sun_webkit_dom_ElementImpl_hasAttributesImpl;
@@ -714,8 +664,6 @@
                Java_com_sun_webkit_dom_ElementImpl_removeAttributeNSImpl;
                Java_com_sun_webkit_dom_ElementImpl_removeAttributeNodeImpl;
                Java_com_sun_webkit_dom_ElementImpl_removeImpl;
-               Java_com_sun_webkit_dom_ElementImpl_scrollByLinesImpl;
-               Java_com_sun_webkit_dom_ElementImpl_scrollByPagesImpl;
                Java_com_sun_webkit_dom_ElementImpl_scrollIntoViewIfNeededImpl;
                Java_com_sun_webkit_dom_ElementImpl_scrollIntoViewImpl;
                Java_com_sun_webkit_dom_ElementImpl_setAttributeImpl;
@@ -761,12 +709,9 @@
                Java_com_sun_webkit_dom_ElementImpl_setOnpasteImpl;
                Java_com_sun_webkit_dom_ElementImpl_setOnresetImpl;
                Java_com_sun_webkit_dom_ElementImpl_setOnscrollImpl;
-               Java_com_sun_webkit_dom_ElementImpl_setOnsearchImpl;
                Java_com_sun_webkit_dom_ElementImpl_setOnselectImpl;
                Java_com_sun_webkit_dom_ElementImpl_setOnselectstartImpl;
                Java_com_sun_webkit_dom_ElementImpl_setOnsubmitImpl;
-               Java_com_sun_webkit_dom_ElementImpl_setOnwebkitfullscreenchangeImpl;
-               Java_com_sun_webkit_dom_ElementImpl_setOnwebkitfullscreenerrorImpl;
                Java_com_sun_webkit_dom_ElementImpl_setScrollLeftImpl;
                Java_com_sun_webkit_dom_ElementImpl_setScrollTopImpl;
                Java_com_sun_webkit_dom_ElementImpl_webkitMatchesSelectorImpl;
@@ -965,7 +910,6 @@
                Java_com_sun_webkit_dom_HTMLDocumentImpl_captureEventsImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_clearImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_closeImpl;
-               Java_com_sun_webkit_dom_HTMLDocumentImpl_getActiveElementImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getAlinkColorImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getBgColorImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getCompatModeImpl;
@@ -973,13 +917,10 @@
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getDirImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getEmbedsImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getFgColorImpl;
-               Java_com_sun_webkit_dom_HTMLDocumentImpl_getHeightImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getLinkColorImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getPluginsImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getScriptsImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_getVlinkColorImpl;
-               Java_com_sun_webkit_dom_HTMLDocumentImpl_getWidthImpl;
-               Java_com_sun_webkit_dom_HTMLDocumentImpl_hasFocusImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_openImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_releaseEventsImpl;
                Java_com_sun_webkit_dom_HTMLDocumentImpl_setAlinkColorImpl;
@@ -999,11 +940,9 @@
                Java_com_sun_webkit_dom_HTMLElementImpl_getDirImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_getDraggableImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_getHiddenImpl;
-               Java_com_sun_webkit_dom_HTMLElementImpl_getInnerHTMLImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_getInnerTextImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_getIsContentEditableImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_getLangImpl;
-               Java_com_sun_webkit_dom_HTMLElementImpl_getOuterHTMLImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_getOuterTextImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_getSpellcheckImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_getTitleImpl;
@@ -1017,11 +956,8 @@
                Java_com_sun_webkit_dom_HTMLElementImpl_setDirImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_setDraggableImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_setHiddenImpl;
-               Java_com_sun_webkit_dom_HTMLElementImpl_setInnerHTMLImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_setInnerTextImpl;
-               Java_com_sun_webkit_dom_HTMLElementImpl_setItemIdImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_setLangImpl;
-               Java_com_sun_webkit_dom_HTMLElementImpl_setOuterHTMLImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_setOuterTextImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_setSpellcheckImpl;
                Java_com_sun_webkit_dom_HTMLElementImpl_setIdImpl;
@@ -1030,7 +966,6 @@
                Java_com_sun_webkit_dom_HTMLElementImpl_setWebkitdropzoneImpl;
                Java_com_sun_webkit_dom_HTMLFieldSetElementImpl_checkValidityImpl;
                Java_com_sun_webkit_dom_HTMLFieldSetElementImpl_getDisabledImpl;
-               Java_com_sun_webkit_dom_HTMLFieldSetElementImpl_getElementsImpl;
                Java_com_sun_webkit_dom_HTMLFieldSetElementImpl_getFormImpl;
                Java_com_sun_webkit_dom_HTMLFieldSetElementImpl_getNameImpl;
                Java_com_sun_webkit_dom_HTMLFieldSetElementImpl_getTypeImpl;
@@ -1132,9 +1067,7 @@
                Java_com_sun_webkit_dom_HTMLHeadElementImpl_setProfileImpl;
                Java_com_sun_webkit_dom_HTMLHeadingElementImpl_getAlignImpl;
                Java_com_sun_webkit_dom_HTMLHeadingElementImpl_setAlignImpl;
-               Java_com_sun_webkit_dom_HTMLHtmlElementImpl_getManifestImpl;
                Java_com_sun_webkit_dom_HTMLHtmlElementImpl_getVersionImpl;
-               Java_com_sun_webkit_dom_HTMLHtmlElementImpl_setManifestImpl;
                Java_com_sun_webkit_dom_HTMLHtmlElementImpl_setVersionImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_getAlignImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_getContentDocumentImpl;
@@ -1145,7 +1078,6 @@
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_getMarginHeightImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_getMarginWidthImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_getNameImpl;
-               Java_com_sun_webkit_dom_HTMLIFrameElementImpl_getSandboxImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_getScrollingImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_getSrcImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_getSrcdocImpl;
@@ -1157,9 +1089,7 @@
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_setMarginHeightImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_setMarginWidthImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_setNameImpl;
-               Java_com_sun_webkit_dom_HTMLIFrameElementImpl_setSandboxImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_setScrollingImpl;
-               Java_com_sun_webkit_dom_HTMLIFrameElementImpl_setSeamlessImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_setSrcImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_setSrcdocImpl;
                Java_com_sun_webkit_dom_HTMLIFrameElementImpl_setWidthImpl;
@@ -1214,10 +1144,8 @@
                Java_com_sun_webkit_dom_HTMLInputElementImpl_getFormNoValidateImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_getFormTargetImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_getHeightImpl;
-               Java_com_sun_webkit_dom_HTMLInputElementImpl_getIncrementalImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_getIndeterminateImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_getLabelsImpl;
-               Java_com_sun_webkit_dom_HTMLInputElementImpl_getListImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_getMaxImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_getMaxLengthImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_getMinImpl;
@@ -1256,7 +1184,6 @@
                Java_com_sun_webkit_dom_HTMLInputElementImpl_setFormNoValidateImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_setFormTargetImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_setHeightImpl;
-               Java_com_sun_webkit_dom_HTMLInputElementImpl_setIncrementalImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_setIndeterminateImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_setMaxImpl;
                Java_com_sun_webkit_dom_HTMLInputElementImpl_setMaxLengthImpl;
@@ -1454,7 +1381,6 @@
                Java_com_sun_webkit_dom_HTMLSelectElementImpl_setAutofocusImpl;
                Java_com_sun_webkit_dom_HTMLSelectElementImpl_setCustomValidityImpl;
                Java_com_sun_webkit_dom_HTMLSelectElementImpl_setDisabledImpl;
-               Java_com_sun_webkit_dom_HTMLSelectElementImpl_setLengthImpl;
                Java_com_sun_webkit_dom_HTMLSelectElementImpl_setMultipleImpl;
                Java_com_sun_webkit_dom_HTMLSelectElementImpl_setNameImpl;
                Java_com_sun_webkit_dom_HTMLSelectElementImpl_setRequiredImpl;
@@ -1622,7 +1548,6 @@
                Java_com_sun_webkit_dom_HTMLUListElementImpl_getTypeImpl;
                Java_com_sun_webkit_dom_HTMLUListElementImpl_setCompactImpl;
                Java_com_sun_webkit_dom_HTMLUListElementImpl_setTypeImpl;
-               Java_com_sun_webkit_dom_KeyboardEventImpl_getAltGraphKeyImpl;
                Java_com_sun_webkit_dom_KeyboardEventImpl_getAltKeyImpl;
                Java_com_sun_webkit_dom_KeyboardEventImpl_getCharCodeImpl;
                Java_com_sun_webkit_dom_KeyboardEventImpl_getCtrlKeyImpl;
@@ -1655,8 +1580,6 @@
                Java_com_sun_webkit_dom_MouseEventImpl_getScreenYImpl;
                Java_com_sun_webkit_dom_MouseEventImpl_getShiftKeyImpl;
                Java_com_sun_webkit_dom_MouseEventImpl_getToElementImpl;
-               Java_com_sun_webkit_dom_MouseEventImpl_getXImpl;
-               Java_com_sun_webkit_dom_MouseEventImpl_getYImpl;
                Java_com_sun_webkit_dom_MouseEventImpl_initMouseEventImpl;
                Java_com_sun_webkit_dom_MutationEventImpl_getAttrChangeImpl;
                Java_com_sun_webkit_dom_MutationEventImpl_getAttrNameImpl;
@@ -1705,7 +1628,6 @@
                Java_com_sun_webkit_dom_NodeImpl_isDefaultNamespaceImpl;
                Java_com_sun_webkit_dom_NodeImpl_isEqualNodeImpl;
                Java_com_sun_webkit_dom_NodeImpl_isSameNodeImpl;
-               Java_com_sun_webkit_dom_NodeImpl_isSupportedImpl;
                Java_com_sun_webkit_dom_NodeImpl_lookupNamespaceURIImpl;
                Java_com_sun_webkit_dom_NodeImpl_lookupPrefixImpl;
                Java_com_sun_webkit_dom_NodeImpl_normalizeImpl;
@@ -1728,8 +1650,6 @@
                Java_com_sun_webkit_dom_NodeListImpl_dispose;
                Java_com_sun_webkit_dom_NodeListImpl_getLengthImpl;
                Java_com_sun_webkit_dom_NodeListImpl_itemImpl;
-               Java_com_sun_webkit_dom_NotationImpl_getPublicIdImpl;
-               Java_com_sun_webkit_dom_NotationImpl_getSystemIdImpl;
                Java_com_sun_webkit_dom_ProcessingInstructionImpl_getSheetImpl;
                Java_com_sun_webkit_dom_ProcessingInstructionImpl_getTargetImpl;
                Java_com_sun_webkit_dom_RGBColorImpl_dispose;
@@ -1814,12 +1734,6 @@
                Java_com_sun_webkit_dom_UIEventImpl_getViewImpl;
                Java_com_sun_webkit_dom_UIEventImpl_getWhichImpl;
                Java_com_sun_webkit_dom_UIEventImpl_initUIEventImpl;
-               Java_com_sun_webkit_dom_WheelEventImpl_getDeltaModeImpl;
-               Java_com_sun_webkit_dom_WheelEventImpl_getWebkitDirectionInvertedFromDeviceImpl;
-               Java_com_sun_webkit_dom_WheelEventImpl_getWheelDeltaImpl;
-               Java_com_sun_webkit_dom_WheelEventImpl_getWheelDeltaXImpl;
-               Java_com_sun_webkit_dom_WheelEventImpl_getWheelDeltaYImpl;
-               Java_com_sun_webkit_dom_WheelEventImpl_initWheelEventImpl;
                Java_com_sun_webkit_dom_XPathExpressionImpl_dispose;
                Java_com_sun_webkit_dom_XPathExpressionImpl_evaluateImpl;
                Java_com_sun_webkit_dom_XPathNSResolverImpl_dispose;
@@ -1871,7 +1785,6 @@
                Java_com_sun_webkit_WCPluginWidget_twkInvalidateWindowlessPluginRect;
                Java_com_sun_webkit_WCPluginWidget_twkSetPlugunFocused;
                Java_com_sun_webkit_WCWidget_initIDs;
-               Java_com_sun_webkit_WatchdogTimer_twkFire;
                Java_com_sun_webkit_WebPage_twkAddJavaScriptBinding;
                Java_com_sun_webkit_WebPage_twkAdjustFrameHeight;
                Java_com_sun_webkit_WebPage_twkBeginPrinting;
